codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2FrameWriter.java

```diff
@@ -623,10 +619,4 @@ private static void verifyErrorCode(long errorCode) {
     private static void verifyWindowSizeIncrement(int windowSizeIncrement) {
         checkPositiveOrZero(windowSizeIncrement, "windowSizeIncrement");
     }
-
-    private static void verifyPingPayload(ByteBuf data) {
```

Comment:
should this change be something that is done outside of this PR ?

Reply:
Some unused code I came across. Want me to make a separate PR for it?

Reply:
Yep as I am almost sure this is also present in 4.1